src/pages/128x64x1/advanced/mixer_setup.c

```diff
@@ -262,21 +262,21 @@ static int expo_row_cb(int absrow, int relrow, int y, void *data)
 
     switch(absrow) {
```

Comment:
Lint Error:
```
258:  Use int16/int64/etc, rather than the C type long  
259:  Use int16/int64/etc, rather than the C type long  
263:  Missing space before ( in switch(  
```